docs/development/intro.rst

```diff
@@ -147,82 +147,103 @@ certainly the same week), are best.
 Get set up
 ==========
 
-.. warning::
+.. note::
+        :class: sphx-glr-download-link-note
 
-    The rest of this page isn't written yet. It's almost identical to
-    https://ctapipe.readthedocs.io/en/latest/developer-guide/getting-started.html so for
-    now, see there. Also, we shouldn't duplicate content from
-    https://docs.astropy.org/en/latest/development/index.html but link
-    there instead.
+        This section is based on the
+        `instructions <https://ctapipe.readthedocs.io/en/latest/developer-guide/getting-started.html>`_
+        for setting up ``ctapipe``.
```

Comment:
Maybe, one can put the documentation link: https://ctapipe.readthedocs.io/en/latest/